### ISSUE TITLE
Text is "burnt" into background

### ISSUE THREAD

**Issue by TraceyK14** [zeit/hyper#1008]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**:  10.11.3 El Capitan
- **Hyper.app version**: 0.8.3
- **Link of a [Gist](https://gist.github.com/) with the contents of your .hyper.js**: https://gist.github.com/TraceyK14/fd6ebb86a6439ac50d0e03966431a3c7
- **Relevent information from devtools** _(CMD+SHIFT+I on Mac OS, CTRL+SHIFT+I elsewhere)_: 

## Issue

When you lose focus on Hyper window and then return focus the text will become "burnt" into the background. 

You can view the text by typing `clear` into the terminal and hitting `enter`. 

Also can be done by reaching the end of the terminal window and continue to press `enter` causing the window to scroll

Hyper's default theme with the exception of the opacity change to view the "burnt" effect.
![2016-11-16_13-44-28](https://cloud.githubusercontent.com/assets/6742945/20361227/d1a6a1ac-ac04-11e6-9c91-06451424f033.png)

My normal terminal theme.
![2016-11-16_13-26-29](https://cloud.githubusercontent.com/assets/6742945/20361230/d4187528-ac04-11e6-88cb-e7f5b94b68c5.png)

**Comment by rauchg** [comment]
Are you using transparency?

**Comment by TraceyK14** [comment]
Yes the opacity on the background is half in the images provided. 
`// terminal background color
    backgroundColor: 'rgba(0,0,0,.5)',`

**Comment by rauchg** [comment]
Yep so the reason I introduced opacity was "optimistic". There are some Electron issues still with transparency :(

**Comment by rauchg** [comment]
And by Electron I potentially mean Chromium as well

**Comment by TraceyK14** [comment]
Yea there was some discussion of it being a potential Electron issue on the slack channel. Do you still want to keep this ticket open until it is resolved on their end? (no telling when or if it will be)

**Comment by ppot** [comment]
@rauchg The macOS transparency has been natively added to electron recently

**Comment by rauchg** [comment]
So, have we tried reproducing this with Hyper + latest electron? Thanks for letting me know

**Comment by ppot** [comment]
@rauchg  No, It was talked today in slack see PR from electron
https://github.com/electron/electron/pull/7898

**Comment by CodeTheory** [comment]
@ppot Vibrancy is not just transparency, right? I think Electron has had transparency for a while with frameless windows like Hyper.app is.

**Comment by ppot** [comment]
@CodeTheory Good question, I can't tell!

**Comment by CodeTheory** [comment]
I believe that's the case. In which case, Electron version shouldn't matter.

**Comment by andydude** [comment]
I'm on Mac OS X 10.10.5 Yosemite, and I have a similar issue, but it might be related to this issue, or the Windows transparency issue, but I'm not sure how to classify it, or even describe it.
<img width="1190" alt="screen shot 2016-11-30 at 9 32 19 pm" src="https://cloud.githubusercontent.com/assets/10523/20779605/8777be8c-b743-11e6-80a4-332b58aa1daa.png">
This first one was after maximizing (the new fullscreen) a window, then minimizing it again.
<img width="1183" alt="screen shot 2016-11-30 at 9 31 47 pm" src="https://cloud.githubusercontent.com/assets/10523/20779606/8777fe2e-b743-11e6-893b-ebafd4d7614d.png">
The second one was opening a new window, then opening a dozen tabs. Notice the time is overwriting the previous time in my prompt.

**Comment by ppot** [comment]
@andydude  What version you have. Since you have the path_helper in you're header it might be related to your version. Did you try the lastest master?

**Comment by andydude** [comment]
Yes, I just tried master, and I have the same issue.
<img width="1220" alt="screen shot 2016-12-01 at 3 08 17 pm" src="https://cloud.githubusercontent.com/assets/10523/20810258/07ec1db8-b7d7-11e6-8adc-383ffc4f624a.png">
<img width="1225" alt="screen shot 2016-12-01 at 3 08 00 pm" src="https://cloud.githubusercontent.com/assets/10523/20810259/07eca5a8-b7d7-11e6-817c-97bfa5e1a74a.png">

**Comment by wmertens** [comment]
If it's any help, this reminds me of an issue Terminal.app had, where if you selected a text shadow with a transparent background, the shadows would stick around when the text moved. AFAICT they "solved" it by removing the text shadow option :)

**Comment by iam4x** [comment]
I have this issue on my MacbookPro Retina 13" touchbar with Sierra last version, but it's not happening with my Hackintosh with a nVidia 980GTX. It might be hardware/graphics/driver related?

**Comment by bsmith-cycorp** [comment]
I'm experiencing this on my Mid 2014 Retina Macbook Pro 15", with Intel Iris Pro 1536 MB graphics. I too have a partly transparent background color, set in the regular config file.

I haven't looked at the code, but perhaps the terminal is getting cleared by "painting" the background color over the existing text? That would cause the text to only get partially faded out if the background is semitransparent, which is exactly the behavior I'm seeing.

**Comment by chabou** [comment]
Seems duplicate #808

**Comment by trueter** [comment]
@albinekb Great news! I'm curious about the nature of the problem. Is there a commit you can reference that specifically addresses this?

**Comment by joeyg-stripe** [comment]
I think I have the same problem as ajcrites
